docs/docs/spark-ddl.md

```diff
@@ -566,3 +566,118 @@ Tags can be removed via the `DROP TAG` sql
 ```sql
 ALTER TABLE prod.db.sample DROP TAG `historical-tag`
 ```
+
+### Iceberg views in Spark
+
+Iceberg views are a [common representation](../../view-spec.md) of a SQL view that aim to be interpreted across multiple query engines.
+This section covers how to create and manage views in Spark using Spark 3.4+ (earlier versions of Spark are not supported).
```

Comment:
may be mention as 3.4 and above, 3.4+ might suggest that above 3.4 (excluding 3.4)